### ISSUE TITLE
Adds infinite space before text

### ISSUE THREAD

**Issue by muuvmuuv** [mblode/vscode-twig-language-2#50]
When I save this html text:

```html
<aside id="aside">
  KEYWORDS
              CATEGORY
</aside>
```

it adds one space every time I save (format) the document before "CATEGORY".

**Comment by mblode** [comment]
That is a very strange bug that is related to the Prettydiff 3 formatter. One solution is to wrap the words in <span> tags. Sorry, abut the ongoing issues!

**Comment by muuvmuuv** [comment]
Yeah, noticed that it is working when wrapping in tags but would be great if this could be fixed. It's really annoying 😄

**Comment by dinukasal** [comment]
have the same issue

**Comment by mblode** [comment]
Labeling this `formatter`. This is one of the prettydiff-rooted formatting bugs — the underlying engine (prettydiff) is unmaintained, so these aren't fixable in isolation. Tracking the migration off prettydiff in #117; subscribe there for progress.

**Comment by mblode** [comment]
Fixed repeated paragraph indentation in 0.12.0. Tests check that formatting twice gives the same result.
